modules.d/90overlay-root/overlay-mount.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# make a read-only nfsroot writeable by using overlayfs
+# the nfsroot is already mounted to $NEWROOT
+# add the parameter rootovl to the kernel, to activate this feature
+
+. /lib/dracut-lib.sh
+
+if ! getargbool 0 rootovl ; then
+    return
+fi
+
+modprobe overlay
+
+# a little bit tuning
+mount -o remount,nolock,noatime $NEWROOT
+
+# Move root
```

Comment:
`mount --make-private /` would fix this. Though I don't know whether that breaks something else.

Reply:
If making `/` private is not an option, please move the pseudo-move into a function